Dockerfile

```diff
@@ -68,22 +64,18 @@ RUN apk add --no-cache \
     python3 \
     python3-dev \
     py3-pip \
-    shellcheck
+    shellcheck \
+    libc6-compat
 
 # Install pre-commit framework
 RUN pip3 install --ignore-installed pre-commit==$PRECOMMIT_VERSION
 
 # Install checkov
 RUN pip3 install --ignore-installed checkov==$CHECKOV_VERSION
 
-# Download Terraform, verify checksum and install to bin dir
-RUN wget \
-    $TERRAFORM_URL \
-    $TERRAFORM_CHECKSUM_URL && \
-    sed -i '/terraform_.*_linux_amd64.zip/!d' $TERRAFORM_CHECKSUM && \
-    sha256sum -cs $TERRAFORM_CHECKSUM && \
-    unzip $TERRAFORM_ARCHIVE -d /usr/local/bin && \
-    rm -f $TERRAFORM_ARCHIVE $TERRAFORM_CHECKSUM
+# Install tfswitch and the Terraform version that is defined in the $TERRAFORM_VERSION build arg
+RUN wget -O- -nv https://raw.githubusercontent.com/warrensbox/terraform-switcher/release/install.sh | sh
+RUN tfswitch $TERRAFORM_VERSION
```

Comment:
The idea is to have a pre-installed version in the image already to avoid having tfswitch installing `Terraform` on every container startup - this is why I decided to install a pre-defined version. In your last review, you requested to review the `--latest-stable` flag from this line - I don't think that we should completely remove it.